### PR TITLE
Bulgarian.java

### PR DESCRIPTION
**Pull request by Rinto-kun**
Bulgarian translation of the Tillerino bot software.

### CODE REVIEW

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java [hunk 1]
```diff
@@ -78,32 +78,32 @@ public void welcomeUser(IRCBotUser user, OsuApiUser apiUser, long inactiveTime)
 	@Override
 	public String unknownCommand(String command) {
 		return "Unknown command \"" + command
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: no translation? other languages translate this

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java [hunk 2]
```diff
@@ -149,29 +149,29 @@ public String help() {
...
-		return "What do you mean nomod with mods?";
+		return "Какво имаш предвид с 'без модове с модове'?";
 	}
 
 	@Override
 	public String outOfRecommendations() {
 		return "[https://github.com/Tillerino/Tillerinobot/wiki/FAQ#the-bot-says-its-out-of-recommendations-what-do"
-				+ " I've recommended everything that I can think of]."
-				+ " Try other recommendation options or use !reset. If you're not sure, check !help.";
+				+ "Препоръчах ти всичко, за което се сетих]."
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: The space in front of `I've` is needed here

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java [hunk 3]
```diff
@@ -234,55 +234,55 @@ public void optionalCommentOnLanguage(IRCBotUser user, OsuApiUser apiUser) {
 		 * P.S. you can put a link to your profile into the line like this:
 		 * [https://osu.ppy.sh/u/2070907 Tillerino]
 		 */
-		user.message("So you like me just the way I am :)");
+		user.message("Хей, благодаря, че смени на български :)");
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: You're not gonna give yourself credit?

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java [hunk 4]
```diff
@@ -149,29 +149,29 @@ public String help() {
...
+		return "Какво имаш предвид с 'без модове с модове'?";
 	}
 
 	@Override
 	public String outOfRecommendations() {
 		return "[https://github.com/Tillerino/Tillerinobot/wiki/FAQ#the-bot-says-its-out-of-recommendations-what-do"
-				+ " I've recommended everything that I can think of]."
-				+ " Try other recommendation options or use !reset. If you're not sure, check !help.";
+				+ "Препоръчах ти всичко, за което се сетих]."
+				+ "Опитай други опции или използвай !reset. Ако не си сигурен, опитай с !help.";
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: same here, should begin with a space

**Comment by omkelderman** [comment]
thanks for the translation! I've left a few comments, could you address those? Although I'm not sure if the two about the spaces are really necessary, since this is done with the "easy way" and this is not suppose to be merge ready and I don't know in what shape @Tillerino this expects to be :P

**Comment by Tillerino** [comment]
@Rinto-kun can you please adress the issues above?

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java (hunk 3 above)
Comment (Rinto-kun) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: No, I don't need any.

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java (hunk 4 above)
Comment (Rinto-kun) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: Got it.

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java (hunk 1 above)
Comment (Rinto-kun) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: Missed it.

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java (hunk 2 above)
Comment (Rinto-kun) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: Okay.

**Comment by Rinto-kun** [comment]
Yes, thank you for the feedback @omkelderman.
These are all my misses.
I'd like to make the changes though I'm completely clueless to github.
Can you instruct me how to proceed?

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java [hunk 5]
```diff
@@ -24,47 +24,47 @@
...
-		return "I'm sorry, I don't know that map. It might be very new, very hard, unranked or not standard osu mode.";
+		return " Съжалявам, не познавам тази карта. Може да е нова, трудна, неодобрена или не направена за стандартен осу! мод. ";
 	}
 
 	@Override
 	public String internalException(String marker) {
-		return "Ugh... Looks like human Tillerino screwed up my wiring."
-				+ " If he doesn't notice soon, could you [https://github.com/Tillerino/Tillerinobot/wiki/Contact inform him]? (reference "
+		return " Ах... Явно човешкият Тилерино ми скапа мрежата."
+				+ "Ако той не забележи скоро, би ли му се [https://github.com/Tillerino/Tillerinobot/wiki/Contact обадил]? (препратка "
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: This one should also start with a space

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java [hunk 6]
```diff
@@ -118,29 +118,29 @@ public String tryWithMods(List<Mods> mods) {
...
-		return "Your complaint has been filed. Tillerino will look into it when he can.";
+		return "Оплакването ти бе изпратено. Тилерино ще го погледне, когато може.";
 	}
 
 	@Override
 	public void hug(final IRCBotUser user, OsuApiUser apiUser) {
-		user.message("Come here, you!");
-		user.action("hugs " + apiUser.getUserName());
+		user.message("Ела насам!");
+		user.action("гушкам" + apiUser.getUserName());
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java]: another missing space

**Comment by omkelderman** [comment]
@Rinto-kun yes of course :smile:

Also please note that I missed some more "needed spaces". I've commented on those for reference.

As for instructions:
* Please go to https://github.com/Rinto-kun/Tillerinobot/edit/patch-1/tillerinobot/src/main/java/tillerino/tillerinobot/lang/Default.java
* On that page make the appropriate changes.

Also one other thing I missed last time: can you please change the filename from `Default.java` to to `Bulgarian.java`. You can do so just above the text-editor: default.java

After that make sure the checkbox "commit directly to ..." is checked (commit directly to...) and click the button "Commit changes".

Last thing we need to know: what is "Bulgarian" translated into Bulgarian? This will be used by people choosing the language (`!set lang ????`)

After this everything should be gud to go :smiley:

**Comment by Rinto-kun** [comment]
@omkelderman 
Thank you again. I'm ready. You say български.
But is it needed to start every new line with a space.
Of course, I corrected those you marked but I left it without spaces where it was the same on the English original. Is that right?

**Comment by Rinto-kun** [comment]
I also noticed one my mistake and did some more corrections to make the bot easier to understand in Bulgarian(didn't touch anything on the code).

**Comment by omkelderman** [comment]
@Rinto-kun 
> But is it needed to start every new line with a space.
> Of course, I corrected those you marked but I left it without spaces where it was the same on the English original. Is that right?

Due to some formatting stuff sometimes a space is needed sometimes not. Doing the same as the original is the way to go.

Anyways, as far as I can tell everything looks fine!
Now the final word is up to @Tillerino.

**Pull request closed without merging**